tests/unit/test_sentry_client.py

```diff
@@ -283,3 +284,170 @@ def test_upload_installable_app_single_chunk_failure(self):
         # Each request can itself be retried 3 times
         # So (1*3)*4 = 16
         assert upload.call_count == 16
+
+    @responses.activate
+    def test_download_artifact_success(self):
+        """Test successful download of artifact."""
+        # Mock HEAD request for file size
+        responses.add(
+            responses.HEAD,
+            "https://example.com/api/0/internal/test-org/test-project/files/preprodartifacts/test-artifact/",
+            headers={"Content-Length": "40"},  # 40 bytes total
+        )
+
+        # Mock GET request for download
+        responses.add(
+            responses.GET,
+            "https://example.com/api/0/internal/test-org/test-project/files/preprodartifacts/test-artifact/",
+            body=b"A" * 20 + b"B" * 20,  # 40 bytes total, will be 2 chunks of 20MB each (but truncated)
```

Comment:
how does this become two chunks?

Reply:
sorry, the comment is not correct. updated/removed